openedx/core/djangoapps/content_libraries/api/blocks.py

```diff
@@ -55,16 +59,27 @@
 from .exceptions import (
     BlockLimitReachedError,
     ContentLibraryBlockNotFound,
+    IncompatibleTypesError,
     InvalidNameError,
     LibraryBlockAlreadyExists,
 )
+from .containers import (
+    create_container,
+    get_container,
+    update_container_children,
+    ContainerMetadata,
+    ContainerType,
+)
 from .libraries import (
     library_collection_locator,
     library_component_usage_key,
     require_permission_for_library_key,
     PublishableItem,
 )
 
+if TYPE_CHECKING:
+    from openedx.core.djangoapps.content_staging.api import StagedContentFileData
```

Comment:
Nit: I think use of `TYPE_CHECKING` is obscure enough that it warrants a comment. I assume this is to avoid a circular dependency?